### ISSUE TITLE
Show the name of Records

### ISSUE THREAD

**Issue by jonaskello** [andrewdavey/immutable-devtools#6]
Example code:

```js
	let MyRecord = Immutable.Record({foo: 4}, "MyRecord");
	let bar = new MyRecord({foo: 3});
	debugger;
```

Printed to the Console:

![image](https://cloud.githubusercontent.com/assets/664076/11588769/4bf0cd3c-9a83-11e5-9a16-eea98eee1947.png)

It would be nice if instead of just printing "Record" the formatter would print the actual name of the record, in this case "MyRecord". Or it may print "Record: MyRecord" to see that is is a Record with this name.

**Comment by andrewdavey** [comment]
Implemented in 0.0.5

**Comment by jonaskello** [comment]
Thanks, this works great!
